lms/templates/problem.html

```diff
@@ -25,42 +25,43 @@ <h3 class="hd hd-2 problem-header" id="${ short_id }-problem-title" aria-describ
        />
       </div>
     % endif
-    <div class="problem-action-buttons-wrapper">
-    % if demand_hint_possible:
-    <span class="problem-action-button-wrapper">
-        <button type="button" class="hint-button problem-action-btn btn-default btn-small" data-value="${_('Hint')}" ${'' if should_enable_next_hint else 'disabled'}><span class="icon fa fa-question" aria-hidden="true"></span>${_('Hint')}</button>
-    </span>
-    % endif
-    % if save_button:
-    <span class="problem-action-button-wrapper">
-        <button type="button" class="save problem-action-btn btn-default btn-small" data-value="${_('Save')}">
-            <span class="icon fa fa-floppy-o" aria-hidden="true"></span>
-            <span aria-hidden="true">${_('Save')}</span>
-            <span class="sr">${_("Save your answer")}</span>
-        </button>
-    </span>
-    % endif
-    % if reset_button:
-    <span class="problem-action-button-wrapper">
-        <button type="button" class="reset problem-action-btn btn-default btn-small" data-value="${_('Reset')}"><span class="icon fa fa-refresh" aria-hidden="true"></span><span aria-hidden="true">${_('Reset')}</span><span class="sr">${_("Reset your answer")}</span></button>
-    </span>
-    % endif
-    % if answer_available:
-    <span class="problem-action-button-wrapper">
-        <button type="button" class="show problem-action-btn btn-default btn-small" aria-describedby="${ short_id }-problem-title"><span class="icon fa fa-info-circle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span></button>
-    </span>
-    % endif
-    </div>
+
     <div class="submit-attempt-container">
       <button type="button" class="submit btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" data-should-enable-submit-button="${ should_enable_submit_button }" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
-          <span class="submit-label" aria-hidden="true">${ submit_button }</span><span class="sr">${_("Submit your answer")}</span>
+          <span class="submit-label" aria-hidden="true">${ submit_button }</span><span class="sr">${_("Submit your answer, some problems have options such as save, reset, hints, or show answer. These options follow the Submit button.")}</span>
```

Comment:
@staubina Nit: punctuate so that Submit your answer is a complete sentence. 
"Submit your answer. Some problems have options such as save, reset, hints, or show answer. These options follow the Submit button."